packages/messaging-api-telegram/src/TelegramClient.ts

```diff
@@ -149,9 +150,14 @@ export default class TelegramClient {
    * @returns True on success.
    */
   setWebhook(url: string, options: SetWebhookOption = {}): Promise<boolean> {
+    const snakecaseOptions = snakeCaseKeys(options);
```

Comment:
Should we do this transform here? Or just apply a deep snake case transform in `this._request`?

Reply:
cool

Reply:
updated